Selection_Algorithm_Repo/Selection_algo_using_Quick_sort.c

```diff
@@ -0,0 +1,70 @@
+#include<stdio.h>
+
+// This uses partition function of QuickSort to find kth smallest element in an unsorted array in nlogk time (avg case)
+// O(n square ) in the worst case
+
+int partition(int * a, int low, int high);
+int selectKth(int * a, int low, int high, int k) {
+
+  if (low == high)
+    return a[low];
+  else {
+
+    // Call the partition to place the first pivot.

```

Comment:
Please fix indentation of this file. It is a 2 space indentation. We are following 4 space one.